lib/reek/smells/utility_function.rb

```diff
@@ -68,7 +68,7 @@ def inspect(ctx)
           context: ctx,
           lines: [ctx.exp.line],
           message: "doesn't depend on instance state (maybe move it to another class?)",
-          parameters: { name: ctx.full_name })]
+          parameters: { name: ctx.name.to_s })]
```

Comment:
Ah, here name was already the same as context. Which makes it superfluous, I think.